src/theme/components/breadcrumb.ts

```diff
@@ -14,6 +16,7 @@ const blogs = {
         color: "dark.ui.typography.heading",
       },
     },
+    _focus: { ...customFocusColor("ui.black", "dark.section.blogs.secondary") },
```

Comment:
Use `dark.ui.typography.heading` for dark mode and keep the focus ring color in alignment with the text color.